core/src/main/resources/AppConstants.properties

```diff
@@ -1,75 +1,92 @@
+## set application.server.type.custom to e.g. 'ACTIVEMQ' or 'TIBCO', to enable additional features.
 SPRING.CONFIG.LOCATIONS=spring${application.server.type}${application.server.type.custom}.xml,springIbisDebuggerAdvice.xml,springCustom.xml
 
-# These files will be loaded in the specified order
+## These files will be loaded in the specified order
 ADDITIONAL.PROPERTIES.FILE=DeploymentSpecifics.properties,BuildInfo.properties,ServerSpecifics_${application.server.type}${application.server.type.custom}.properties,SideSpecifics_${dtap.side}.properties,StageSpecifics_${dtap.stage}.properties,Test.properties
-#Deprecated cannot/should not be used anymore
+## [Deprecated] should not be used anymore
 ADDITIONAL.PROPERTIES.FILE.SUFFIX=
 
-# If started from a container, this property will automatically be determined/set during startup
+## If started from a container, this property will automatically be determined/set during startup
 application.server.type=
+
 application.server.type.custom=${transactionmanager.type.default}
-# the transactionmanager.type.default property allows an overridable default to be set by a hosting platform
+
+## the transactionmanager.type.default property allows an overridable default to be set by a hosting platform
 transactionmanager.type.default=
 
-# If TRUE, authentication is enforced on ALL servlets
-# When left empty the default is TRUE, and FALSE when dtap.stage == LOC
+## If TRUE, authentication is enforced on ALL servlets
+## When left empty the default is TRUE, and FALSE when dtap.stage == LOC
 application.security.http.authentication=
-# Either NONE or CONFIDENTIAL. When set to CONFIDENTIAL a secure protocol (SSL/TLS/HTTPS) is enforced for ALL servlets
-# When left empty the default is CONFIDENTIAL, and NONE when dtap.stage == LOC
+
+## Either NONE or CONFIDENTIAL. When set to CONFIDENTIAL a secure protocol (SSL/TLS/HTTPS) is enforced for ALL servlets
+## When left empty the default is CONFIDENTIAL, and NONE when dtap.stage == LOC
 application.security.http.transportGuarantee=
 
-# Authenticators for HTTP endpoints
-# application.security.http.authenticators=myADAuth
-# application.security.http.authenticators.myADAuth.type=AD
-# application.security.http.authenticators.myADAuth.baseDn=DC=company,DC=org
-# application.security.http.authenticators.myADAuth.url=ldap://10.1.2.3
+## Authenticators for HTTP endpoints
+#application.security.http.authenticators=myADAuth
+#application.security.http.authenticators.myADAuth.type=AD
+#application.security.http.authenticators.myADAuth.baseDn=DC=company,DC=org
+#application.security.http.authenticators.myADAuth.url=ldap://10.1.2.3
 application.security.http.authenticators=
 
 application.name=IAF
-#Deprecated as we now use maven this can be determined dynamically
+
+## [Deprecated] Deprecated as we now use maven this can be determined dynamically
 application.version=${frankframework-core.version}
```

Comment:
```suggestion
## [Deprecated] [Generated] Deprecated as we now use maven this will be determined dynamically during the build pipeline.
application.version=${frankframework-core.version}
```